plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/functions/tablechanges/TableChangesSplitSource.java

```diff
@@ -113,6 +113,7 @@ public CompletableFuture<ConnectorSplitBatch> getNextBatch(int maxSize) // TODO
                 for (DeltaLakeTransactionLogEntry entry : entries) {
                     if (entry.getAdd() != null && entry.getAdd().isDataChange()) {
                         AddFileEntry addEntry = entry.getAdd();
+                        // paths can be absolute as well in case of shallow-cloned tables
```

Comment:
Please add test about `table_changes` function with shallow-cloned table.

Reply:
@ebyhr I have added the test in TestDeltaLakeCloneTableCompatibility itself, please let me know If I need to separate this